deeplake/auto/structured/dataframe.py

```diff
@@ -38,24 +36,28 @@ def fill_dataset(self, ds: Dataset, use_progress_bar: bool = True) -> Dataset:
         iterator = tqdm(
             keys,
             desc="Ingesting... (%i keys skipped)" % (len(skipped_keys)),
-            disable=not use_progress_bar,
+            disable=not progressbar,
         )
         with ds, iterator:
             for key in iterator:
+                if progressbar:
+                    print(f"\nkey={key}, dtype={self.source[key].dtype}")
                 try:
                     dtype = self.source[key].dtype
                     if dtype == np.dtype("object"):
-                        self.source[key].fillna("", inplace=True)
-                        ds.create_tensor(key, dtype=str, htype="text")
+                        if key not in ds.tensors:
```

Comment:
Allows ingesting multiple parquete files to one dataset. Note that we don't use `exist_ok` arg of `create_tensor` on purpose.. user might want dataset dtype to be different from dataframe dtype, specify compression etc.